invoke.go

```diff
@@ -82,7 +82,7 @@ func (s *Scope) Invoke(function interface{}, opts ...InvokeOption) error {
 		s.isVerifiedAcyclic = true
 	}
 
-	args, err := pl.BuildList(s)
+	args, err := pl.BuildList(s, false)
```

Comment:
```suggestion
	args, err := pl.BuildList(s, false /* decorating */)
```
per https://github.com/uber-go/guide/blob/master/style.md#avoid-naked-parameters